README.md

```diff
@@ -21,22 +20,23 @@ Chcete-li nám pomáhat na dalším rozvoji projektu, vyplňte formulář na [ce
 - Mac Catalyst (TODO)
 - Apple Watch (9+)
 
-# Hacking
+# Contributing
 
-> Jestli chceš přidat nové funkce, nebo máš nápady tak se připoj k nám na Slack.
+At first if you want to add new features or have ideas, join us on Slack.
 
-Základní použití je jednoduché:
+Basic usage is simple:
 
 ```bash
 git clone git@github.com:cesko-digital/movapp-apple.git movapp-apple
 cd movapp-apple
 ```
 
-Další informace najdete v souboru [CONTRIBUTING](CONTRIBUTING.md).
+Further information can be found in the [CONTRIBUTING](CONTRIBUTING.md) file.
 
-## Podklady
+## Usefull links
 
 - [Scope of Work](https://docs.google.com/document/d/1r7OdwaUiGZKd2qcHHMGeqtTYHlOB6zOzfW9K1sxuY6s/edit?usp=sharing)
-- [Dočasná grafika](https://www.sketch.com/s/5d264dba-950d-4169-8a59-9c048c9183a5/a/3OwLoqG/play)
-- [Aktuální sada screenshotů](https://cesko-digital.github.io/movapp-apple/screenshots.html)
+- [Temporary Graphics](https://www.sketch.com/s/5d264dba-950d-4169-8a59-9c048c9183a5/a/3OwLoqG/play)
```

Comment:
@met this should not be used right, we have "changed" design in figma?

Reply:
@pionl Máme figmu pro některé nové věci, ale pro ty staré základní jsme tuším zatím figmu nevytvořili. Což je vlastně dobrá přípomínka.